app/src/components/Home/components/EmptyCard/index.tsx

```diff
@@ -36,7 +38,10 @@ export const HomepageEmptyCard: React.FC<Props> = ({
             <div className="homepage-emptycard-icon">{icon}</div>
           )}
           <div className="header-content">
-            <h1 className="homepage-emptycard-title">{title}</h1>
+            <div className="title-badge-container">
+              <h1 className="homepage-emptycard-title">{title}</h1>
+              {title === "API client" && <RQBadge badgeText="BETA" className="beta-badge" />}
```

Comment:
No need to pass `className`, also directly target `rq-bagde` class from parent container ie use 
```css
.title-badge-container .rq-badge {}